targettests/Kernel/signature-0.cpp

```diff
@@ -0,0 +1,121 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2023 NVIDIA Corporation & Affiliates.                  *
```

Comment:
Nit: new files should probably have the right copyright date, though we should probably revisit how to handle these as a team.